### PR TITLE
[WCM] Wallet loading screen

### PR DESCRIPTION
**Pull request by radeknovis**

### CODE REVIEW

File: Sources/WalletConnectModal/Modal/Screens/GetAWalletView.swift [hunk 1]
```diff
@@ -67,6 +66,8 @@ struct GetAWalletView: View {
     }
 }
 
+#if DEBUG
```
Comment (flypaper0) [inline Sources/WalletConnectModal/Modal/Screens/GetAWalletView.swift:69]: Could you please double check that previews is wrapped in DEBUG everywhere?

File: Sources/WalletConnectModal/Modal/Screens/WalletList.swift [hunk 2]
```diff
@@ -188,50 +185,15 @@ struct WalletList: View {
                 $0.onTapGesture {
                     withAnimation {
                         navigateTo(.walletDetail(wallet))
+                        
+                        DispatchQueue.main.asyncAfter(deadline: .now() + 0.5) {
```
Comment (flypaper0) [inline Sources/WalletConnectModal/Modal/Screens/WalletList.swift:189]: why we need delay?

**Review by flypaper0 — APPROVED** [review]

File: Sources/WalletConnectModal/Modal/Screens/WalletList.swift (hunk 2 above)
Comment (radeknovis) [reply Sources/WalletConnectModal/Modal/Screens/WalletList.swift:189]: for slightly better UX, users will see `Continue in WalletXYZ` briefly and then it actually performs the deeplink

Comment (radeknovis) [reply Sources/WalletConnectModal/Modal/Screens/WalletList.swift:189]: left comment explaining this

File: Sources/WalletConnectModal/Modal/Screens/GetAWalletView.swift (hunk 1 above)
Comment (radeknovis) [reply Sources/WalletConnectModal/Modal/Screens/GetAWalletView.swift:69]: Added where it was missing

**Pull request merged**